packages/embarkjs-ens/src/ENSFunctions.js

```diff
@@ -100,10 +100,8 @@ function resolveName(name, ens, createResolverContract, callback, _namehash) {
   });
 }
 
-if (typeof module !== 'undefined' && module.exports) {
-  module.exports = {
-    registerSubDomain,
-    resolveName,
-    lookupAddress
-  };
+export default {
+  registerSubDomain,
+  resolveName,
+  lookupAddress
 }
```

Comment:
We need to make sure this works both for the ENS deploy with `embark run` and inside the Dapp, as well as checking if using the Node embarkjs.js script works.